packages/server/src/server.ts

```diff
@@ -210,7 +210,10 @@ const logAndHandleResponse = async (methodName: any, methodParams: any, methodFu
 
     const response = await methodFunction(requestIdPrefix);
     if (response instanceof JsonRpcError) {
-      logger.error(`${requestIdPrefix} ${response.message}`);
+      // log only if it is not a contract revert
+      if (!(response.code === predefined.CONTRACT_REVERT().code)) {
```

Comment:
yes, this is preventing logs from `CONTRACT_REVERT` since they are not actual errors.
What we could do, is keep logging `CONTRACT_REVERT` responses here, but as Debug or Trace.

Reply:
debug trace sounds good

Reply:
✅  done!